templates/part.eventsrepeat.php

```diff
@@ -54,13 +54,13 @@ class="pull-right pull-half"
 	</label>
 	<div class="pull-right pull-half">
```

Comment:
if I remove .pull-half it looks like this .... a bit better from my pov and consumes less or equal space

![screenshot from 2018-06-14 21-01-12](https://user-images.githubusercontent.com/1005065/41432398-4e2923ae-7016-11e8-8768-89591f060ed5.png)

@Julian1998 what do you think?

Reply:
I will update this PR in some days... ;)